### ISSUE TITLE
False positive (error) on multiple comments during a line continuation

### ISSUE THREAD

**Issue by aames** [RedCoolBeans/dockerlint#42]
### Issue details

In my Dockerfile I have my ENV instructions reduced to 1 layer. (per the example at: https://docs.docker.com/engine/reference/builder/#env) 

I have some comments during a line continuation using #

### Steps to reproduce/test case

Dockerfile with multiple env instructions on one layer. 
Specifically 3 comment lines in a row, during a line continuation.

Here is a Dockerfile I am able to reproduce this with: 
```
FROM centos
ENV FOO=fooo \
    BAR=/opt/bar \
    BAS=elk \
# 
#  Baz does baz things
#
    BAZ=/opt/baz\ /r \
    BAN=/opt/ban
```
### Affected version(s)

dockerlint@0.3.1
